services/src/main/java/com/linkedin/kmf/services/configs/ProduceServiceConfig.java

```diff
@@ -57,16 +59,19 @@
   public static final String REBALANCE_ENABLED_DOC = "Periodically move leaders and replica assignments so they are "
       + "evenly spread amongst brokers.";
 
-  public static final String REBALANCE_PARTITION_MULTIPLE_CONFIG = "produce.topic.rebalancePartitionMultiple";
-  public static final String REBALANCE_PARTITION_MULTIPLE_DOC = "Determines the number of partitions per broker in the ideal case.";
+  public static final String REBALANCE_PARTITION_FACTOR_CONFIG = "produce.topic.rebalancePartitionFactor";
+  public static final String REBALANCE_PARTITION_FACTOR_DOC = "Determines the number of partitions per broker in the ideal case.";
 
   public static final String REBALANCE_THRESHOLD_CONFIG = "produce.topic.rebalanceThreshold";
   public static final String REBALANCE_THRESHOLD_DOC = "Determines the number of partitions per broker in the ideal case.";
 
+  public static final String REBALANCE_DELAY_MS_CONFIG = "produce.topic.rebalanceDelayMs";
```

Comment:
Is there any reason that user wants to explicitly config this value? If not, maybe we can simply provide a default value without exposing this as a config.